app.sample.json

```diff
@@ -0,0 +1,13 @@
+{
+    "expo": {
+        "extra": {
+            "IDPS": ["read.test"],
+            "LANGUAGE_CODE": "?"
+        },
+        "revisionId": "?",
```

Comment:
I don't know what this currently is set to. It could be a GIT commit revision or anything that identifies a change. Perhaps to simplify this make it a value like the string `"1"`.